.github/CONTRIBUTING.md

```diff
@@ -10,6 +10,17 @@
 * Clone your version on your account down to your machine from your account, e.g,. `git clone https://github.com/<yourgithubusername>/ckanr.git`
 * Make sure to track progress upstream (i.e., on our version of `ckanr` at `ropensci/ckanr`) by doing `git remote add upstream https://github.com/ropensci/ckanr.git`. Before making changes make sure to pull changes in from upstream by doing either `git fetch upstream` then merge later or `git pull upstream` to fetch and merge in one step
 * Make your changes (bonus points for making changes on a new feature branch)
+* Test your changes locally. You can spin up a local CKAN site running at <http://localhost:5000/> for testing with
```

Comment:
Docker installation instructions might be useful here.

Reply:
Good catch! I think just pointing to https://docs.docker.com/compose/install/ is enough, right?